packages/spruce-next-helpers/src/skillskit/next/_document.js

```diff
@@ -56,20 +56,22 @@ export default class MyDocument extends Document {
 				? `skill-${publicRuntimeConfig.SLUG}`
 				: ''
 
-		const { renderLocation } = this.props
+		const { renderLocation, legacyStylesheet } = this.props
 
 		return (
 			<html className={`skill ${bodyClassName}`}>
 				<Head>
 					<meta name="viewport" content="width=device-width, initial-scale=1" />
-					{publicRuntimeConfig && publicRuntimeConfig.SKILL_STYLESHEET && (
-						<link
-							href={publicRuntimeConfig.SKILL_STYLESHEET}
-							rel="stylesheet"
-							type="text/css"
-							charSet="UTF-8"
-						/>
-					)}
+					{legacyStylesheet &&
```

Comment:
This shouldn't be in `_document.js`, since _theoretically_ you could client-side nav between a legacy and a heartwood page. It should move over to `_app.js`, which is rendered per-page client-side